system-int-test/test/cmr/system_int_test/search/acls/collection_test.clj

```diff
@@ -360,7 +360,7 @@
                                     :collection {:token guest-token
                                                  :page-size 100
                                                  :concept-id (:concept-id coll5)})]
-        (is (= hits (count (:entries results))))
+        (is (= (bigint hits) (count (:entries results))))
```

Comment:
The bigint casts may not be necessary since Clojure will default to Long. Would like to see a test that forces exceeds the Integer/MAX_VALUE

Reply:
Going to opt for no automatic test in the test suite for this due to concerns over time and resource constraints